cloudbuild.yaml

```diff
@@ -285,7 +285,7 @@ steps:
         set -o pipefail
         pids=()
         cloudProducts=("generic" "gke-autopilot")
-        declare -A versionsAndRegions=( [1.26]=asia-east1 [1.27]=us-east1 [1.28]=us-west1 )
+        declare -A versionsAndRegions=( [1.27]=us-east1 [1.28]=us-west1 [1.29]=us-central1 )
```

Comment:
```suggestion
        declare -A versionsAndRegions=( [1.27]=us-east1 [1.28]=us-west1 [1.29]=asia-east1 )
```

Reply:
changed the region to asia-east1 for 1.29